www/security_checks.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/**
+ * Parse the page data to find the score for security headers
+ *
+ * @return array An array with security headers list and a total score or empty array
+ */
+function getSecurityHeadersInfo($testInfo, $testRunResults) {
+  $testResults = $testRunResults->getstepResult(1);
+  if ($testResults) {
+    $securityHeaders = $testResults->getRawResults()['securityHeaders'];
+  }
+
+  return $securityHeaders;
+}
+
+/**
+ * Parse the page data to find any JavaScript vulnerabilities detected
+ * in libraries on the page
+ *
+ * @return boolean A array of vulnerabilities count detected by their severity
+ */
+function getSecurityVulnerabilitiesBySeverity($testInfo, $testRunResults) {
+  $testResults = $testRunResults->getstepResult(1);
```

Comment:
@pmeenan this part worked for me on my local env but I wasn't 100% sure that it will always work with the different settings that can be chosen by the user when doing a test. Just thought I'd ping you on it :-)

Reply:
Yep, that should be fine. I don't think any of the UI deals with showing grades for multiple steps in a multi-step flow.

Reply:
Cool, thanks for confirming 👌